packages/lodestar/test/e2e/doppelganger/doppelganger.test.ts

```diff
@@ -0,0 +1,301 @@
+import chai, {expect} from "chai";
+import chaiAsPromised from "chai-as-promised";
+import {phase0, ssz} from "@chainsafe/lodestar-types";
+import {IChainConfig} from "@chainsafe/lodestar-config";
+import {createAttesterDuty, getAndInitDevValidators} from "../../utils/node/validator";
+import {ChainEvent} from "../../../src/chain";
+import {Network} from "../../../src/network";
+import {connect} from "../../utils/network";
+import {testLogger, LogLevel, TestLoggerOpts} from "../../utils/logger";
+import {getDevBeaconNode} from "../../utils/node/beacon";
+import {waitForEvent} from "../../utils/events/resolver";
+import {fromHexString} from "@chainsafe/ssz";
+import {generateAttestationData} from "@chainsafe/lodestar-beacon-state-transition/test/utils/attestation";
+import {SLOTS_PER_EPOCH} from "@chainsafe/lodestar-params";
+
+chai.use(chaiAsPromised);
+
+/* eslint-disable @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-assignment */
+describe("doppelganger / doppelganger test", function () {
+  const afterEachCallbacks: (() => Promise<unknown> | void)[] = [];
+  afterEach(async () => {
+    while (afterEachCallbacks.length > 0) {
+      const callback = afterEachCallbacks.pop();
+      if (callback) await callback();
+    }
+  });
+
+  const validatorCount = 1;
+  const SECONDS_PER_SLOT = 2;
+  const genesisSlotsDelay = 5;
+  const beaconParams: Pick<IChainConfig, "SECONDS_PER_SLOT"> = {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    SECONDS_PER_SLOT: SECONDS_PER_SLOT,
```

Comment:
You can just do `SECONDS_PER_SLOT: 2` as you reference `beaconParams.SECONDS_PER_SLOT` below